common/determined_common/experimental/model.py

```diff
@@ -205,6 +205,9 @@ def to_json(self) -> Dict[str, Any]:
             "metadata": self.metadata,
         }
 
+    def __repr__(self) -> str:
+        return "Model(name={})".format(self.name)
```

Comment:
What do you think about printing metadata fields?

Reply:
I was 50/50 on this. Do you think we should pretty print it as JSON? So the model could have multiple lines of output?

Reply:
i'm in favor of pretty print json